tests/tests.cpp

```diff
@@ -0,0 +1,161 @@
+#include <catch2/catch.hpp>
+#include "json/json.h"
+
+#include <llvm/IR/Module.h>
+#include <llvm/IR/LLVMContext.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/Support/SourceMgr.h>
+#include <llvm/IRReader/IRReader.h>
+
+#include "value_relations_plugin.hpp"
+#include "dg/llvm/ValueRelations/getValName.h"
+
+#include <string>
+#include <iostream>
+#include <cstdio>
+#include <fstream>
+
+bool fileExists(const std::string& path) {
+    std::ifstream f(path);
+    return f.good();
+}
+
+std::string compileBenchmark(const std::string& path, const std::string& benchmark) {
+    std::string targetFile = "tmp/" + benchmark + ".ll";
+
+    if (! fileExists(targetFile)) {
+        std::string command = "clang-10 -S -emit-llvm " + path + "/" + benchmark + ".i -o " + targetFile;
```

Comment:
Clang version should be configured by the build system to use version compatible with LLVM that this project was compiled with.

Reply:
Well, I solved this, but the solution feels pretty hacked.